test/integration/patients/worklist/schedule.js

```diff
@@ -766,7 +780,81 @@ context('schedule page', function() {
 
     cy
       .url()
-      .should('contain', 'flow/1/action/2');
+      .should('contain', 'schedule');
+
+    cy
+      .server()
+      .routePatient(fx => {
+        fx.data.id = '1';
+        fx.data.attributes.first_name = 'First';
+        fx.data.attributes.last_name = 'Last';
+
+        return fx;
+      })
+      .routePatientActions(_.identity, '2')
+      .routePatientFlows(_.identity, '2')
+      .visit('/patient/dashboard/1')
+      .wait('@routePatient')
+      .wait('@routePatientActions')
+      .wait('@routePatientFlows');
+
+    cy
+      .get('.patient__context-trail')
+      .should('contain', 'First Last');
+
+    cy
+      .server()
+      .routePatient(fx => {
+        fx.data.id = '1';
+
+        return fx;
+      })
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.state = { data: { id: '55555' } };
+
+        return fx;
+      })
+      .routePatientActions(_.identity, '1')
+      .routePatientFlows(_.identity, '1')
+      .routeActionActivity()
+      .routePrograms()
+      .routeAllProgramActions()
+      .routeAllProgramFlows()
+      .routeFlow()
+      .visit('/patient/1/action/1')
+      .wait('@routePatientActions')
+      .wait('@routePatient')
+      .wait('@routeAction');
+
+    cy
+      .get('.patient__layout')
+      .find('.patient__tab--selected')
+      .contains('Data & Events');
+
+    cy
+      .server()
+      .routeFlow()
+      .routeFlowActions()
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.attributes.name = 'Test Action';
+
+        fx.data.relationships.flow = { data: { id: '1' } };
+
+        return fx;
+      })
+      .routePatientByFlow()
+      .routeActionActivity()
+      .visit('/flow/1/action/1')
+      .wait('@routeFlow')
+      .wait('@routePatientByFlow')
+      .wait('@routeFlowActions');
+
+    cy
+      .get('.sidebar')
+      .find('[data-name-region] .action-sidebar__name')
+      .should('contain', 'Test Action');
   });
```

Comment:
For the last three sections of this test where I verify that the reduced clinician can load the patient/flow pages properly, do I need to load all this route fixture data? Or can some of it be stripped out?

It seems like a lot of data to load since I'm only tested one small thing on each page.

Reply:
yeah you probably need all of that data..  I _think_ you may need to wrap each of the `visit`s in it's own `context` too.  I forget exactly but i think the `visit` call clears the coverage.